std/net/curl.d

```diff
@@ -4300,7 +4352,7 @@ struct Curl
                 interleavefunction, chunk_data, chunk_bgn_function,
                 chunk_end_function, fnmatch_data, fnmatch_function, cookiejar, postfields);
 
-            foreach (option; tt)
+            static foreach (option; tt)
```

Comment:
I would generally advise against mixing different kinds of changes in one commit.